src/BuildScriptGenerator/Node/NodeDependencyChecker.cs

```diff
@@ -30,10 +30,11 @@ public NodeDependencyChecker(ILogger<NodeDependencyChecker> logger)
 
         public IEnumerable<ICheckerMessage> CheckSourceRepo(ISourceRepo repo)
         {
-            dynamic packageJson = NodePlatform.GetPackageJsonObject(repo, null);
+            dynamic packageJson = NodePlatform.GetPackageJsonObject(repo, _logger);
```

Comment:
This was causing exceptions to be thrown because `GetPackageJsonObject` tries writing to a log.